Applications/spire/include/spire/book_view/book_view_properties_dialog.hpp

```diff
@@ -35,7 +35,7 @@ namespace Spire {
         Qt::WindowFlags flags = 0);
 
       //! Returns the properties represented by this dialog.
-      BookViewProperties get_properties() const;
+      BookViewProperties get_properties();
```

Comment:
This should be const.

Reply:
I don't think this can be const because the m_random_engine() call isn't const.

Reply:
There is no use of the random generator here.

Reply:
whoops, embarassing...